tests/import_test.go

```diff
@@ -264,6 +264,25 @@ var _ = Describe("Importer Test Suite-Block_device", func() {
 		Expect(same).To(BeTrue())
 
 	})
+
+	It("Should create blank raw image for block PV", func() {
+		if !f.IsBlockVolumeStorageClassAvailable() {
+			Skip("Storage Class for block volume is not available")
+		}
+		dv := utils.NewDataVolumeForBlankRawImage("create-blank-image-to-block-pvc", "500Mi")
```

Comment:
So NewDataVolumeForBlankRawImage doesn't appear to set block volume mode. So that should not trigger the event and this test should fail.

Reply:
Maybe add a NewDataVolumeForBlankRawImageBlock that takes storage class and volume mode parameters.

Reply:
yea... I missed that.

Reply:
Done